test_conformance/basic/test_progvar.cpp

```diff
@@ -1553,6 +1600,18 @@ int test_progvar_prog_scope_uninit(cl_device_id device, cl_context context, cl_c
 // Test support for variables at program scope. Initialized data.
 int test_progvar_prog_scope_init(cl_device_id device, cl_context context, cl_command_queue queue, int num_elements)
 {
+    cl_bool skip{ CL_FALSE };
+    auto error = should_skip(device, skip);
+    if (CL_SUCCESS != error)
+    {
+        return TEST_FAIL;
+    }
+    if (skip)
+    {
+        log_info("Skipping progvar_prog_scope_misc since it is optionally not "
+                 "supported on this device");
```

Comment:
Please add `\n` in the end of `log_info` message

Reply:
Fixed.